### ISSUE TITLE
Weighting samples

### ISSUE THREAD

**Issue by meilame-tayebjee** [InseeFrLab/codif-ape-train#38]
In the refactored version, we add the true labels of each NAF code to the training set. 
These 732 sentences are the golden standard, we want the model to be perfect on them: they weigh more.

Look at sample wieghting : https://stackoverflow.com/questions/66374709/adding-custom-weights-to-training-data-in-pytorch
